WindowManager/README.md

```diff
@@ -118,27 +118,28 @@ that might occur when your layout is inflated. If you are looking for specific
 information for laying out views you should get the width/height from the
 `Configuration` object or the `DecorView`.
 
-To access these APIs, you can use, when working in Kotlin, a `WindowInfoRepo`:
+To access these APIs, you can use, when working in Kotlin, a `WindowInfoRepository`:
 
 ``` java
-val windowInfoRepo = windowInfoRepository()
+val windowInfoRepository = windowInfoRepository()
 ```
 
-From here you now have access to the WindowMetrics APIs and can easily call
+From here you now have access to the WindowMetrics APIs collecting events from it Flow:
 
 ``` java
-windowInfoRepo.currentWindowMetrics
-windowInfoRepo.maximumWindowMetrics
+windowInfoRepository.currentWindowMetrics
+windowInfoRepository.maximumWindowMetrics
```

Comment:
I believe maximum window metrics is removed from themWindowInfoRepository API

Reply:
Thanks!

I updated the WindowMetrics section correcting this and adding a note on the available callback adapter's methods.